main/src/org/destinationsol/game/sound/OggSoundUrnList.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.destinationsol.game.sound;
+
+
+import org.destinationsol.common.SolMath;
+
+import java.util.List;
+
+/**
+ * Represents a set of random OggSound urns with a single basePitch assigned to each of them.
+ * <p>
+ * This as an alternative to sounds being randomly fetched from a specified folder -
+ * a workflow that wouldn't be viable with gestalt.
+ */
+public class OggSoundUrnList {
```

Comment:
In the original asset system, "select a folder and fetch a random sound file from it" is a common pattern. It doesn't quite work with gestalt - theoretically, regex-based asset fetching could be implemented (_from all the available sound assets fetch the ones starting with "burning"..._) - but that sounds like a terrible idea in terms of performance.

This helper class is an alternative solution - it stores a list of sound Urns that OggSoundManager randomly selects a Urn from when prompted.